plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/TrinoS3ClientFactory.java

```diff
@@ -109,7 +108,7 @@ private AmazonS3 createS3Client(Configuration config)
         AmazonS3Builder<? extends AmazonS3Builder<?, ?>, ? extends AmazonS3> clientBuilder = AmazonS3Client.builder()
                 .withCredentials(awsCredentialsProvider)
                 .withClientConfiguration(clientConfiguration)
-                .withMetricsCollector(new TrinoS3FileSystemMetricCollector(TrinoS3FileSystem.getFileSystemStats()))
```

Comment:
Why not `METRIC_COLLECTOR` from `TrinoS3FileSystem` ?

Maybe this is a code duplication consequence - see https://github.com/trinodb/trino/issues/14348 - or maybe it is intentional. In any case it is worth digging a bit deeper here.